src/components/common/Dialogs/DomainFundSelectorSection/DomainFundSelectorSection.tsx

```diff
@@ -38,38 +38,24 @@ const DomainFundSelectorSection = ({
   transferBetweenDomains,
   disabled,
 }: Props) => {
-  const {
-    watch,
-    setValue,
-    formState: { errors },
-    clearErrors,
-  } = useFormContext();
-  const { motionDomainId, fromDomain, toDomain } = watch();
+  const { watch, setValue, trigger } = useFormContext();
+  const { motionDomainId } = watch();
+
   const handleFromDomainChange = (fromDomainValue) => {
     if (motionDomainId !== fromDomainValue) {
       setValue('motionDomainId', fromDomainValue);
     }
+    trigger();
```

Comment:
Consider passing: `['fromDomainId', 'toDomainId']` as the argument to `trigger` so it only validates those fields, because currently (in a colony generated via the script) if you switch the `to` domain to the same domain as the `from` domain, but then change the `from` domain so they're different again, the validation message for the amount input is displayed, even though it hasn't been touched.

Reply:
I thought about that, and I think it's necessary for the amount to get revalidated if the from domain changes

Say you have 2 domains: A with 5 tokens and B with 1 token
If you select domain A and enter an amount of 2 tokens, it works okay. But when you change the domain to B, unless you trigger the amount validation, you won't see an error (and there's not enough balance in B for 2 tokens)

Maybe the problem is that we show the error for fields that haven't been touched. I'll look into it